<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QTabWidget" name="topTabWidget">
      <property name="styleSheet">
       <string notr="true">#topTabWidget::tab-bar {
  alignment: left;
}

#topTabWidget::pane {
  padding:0;
  border:none;
}</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="configTab">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/config.png</normaloff>:/icon/config.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="NConfigWidget" name="nConfigWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="codeTab">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/code.png</normaloff>:/icon/code.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="NCodeWidget" name="nCodeWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutTab">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/layout.png</normaloff>:/icon/layout.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="NLayoutWidget" name="nLayoutWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="imageTab">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/image.png</normaloff>:/icon/image.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="NImageWidget" name="nImageWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveAll"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAddScene">
   <property name="text">
    <string>AddScene</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NConfigWidget</class>
   <extends>QWidget</extends>
   <header>n_config_widget.h</header>
   <container>1</container>
   <slots>
    <slot>saveConfig()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>NCodeWidget</class>
   <extends>QWidget</extends>
   <header>n_code_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NImageWidget</class>
   <extends>QWidget</extends>
   <header>n_image_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLayoutWidget</class>
   <extends>QWidget</extends>
   <header>n_layout_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNewProject</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenProject</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveAll</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newProject()</slot>
  <slot>openProject()</slot>
  <slot>saveAll()</slot>
 </slots>
</ui>
